backend/server.js

```diff
@@ -2,11 +2,48 @@ import express from 'express'
 import bodyParser from 'body-parser'
 import cors from 'cors'
 import mongoose from 'mongoose'
+import crypto from 'crypto'
+import bcrypt from 'bcrypt-nodejs'
 
 const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/authAPI"
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
 mongoose.Promise = Promise
 
+const User = mongoose.model('User', {
+  name: {
+    type: String,
+    unique: true,
+    minlength: 3
+  },
+  email: {
+    type: String,
+    unique: true
+  },
```

Comment:
It's interesting that we've generally adapted the idea that people can have identical names but not duplicate email addresses